### ISSUE TITLE
BUSCO

### ISSUE THREAD

**Issue by flavioau** [ubccr/software-layer#91]
Hello, would you be able to install BUSCO?
https://busco.ezlab.org/

Thank you

**Comment by tonykew** [comment]
busco is now available in the ccrsoft/2023.01 software environment:

```
vortex1$ module spider busco

-------------------------------------------------------------------------------------------------------------------------------------------------------------------
  busco: busco/5.4.3
-------------------------------------------------------------------------------------------------------------------------------------------------------------------
    Description:
      BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs

    Properties:
      Bioinformatic libraries/apps

    You will need to load all module(s) on any one of the lines below before the "busco/5.4.3" module is available to load.

      gcc/11.2.0  openmpi/4.1.1
 
    Help:
      
      Description
      ===========
      BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs
      
      
      More information
      ================
       - Homepage: https://busco.ezlab.org/
      

vortex1$ 
```

e.g.


`$ module load gcc/11.2.0 openmpi/4.1.1 busco`

**Comment by tonykew** [comment]
Usage notes:

BUSCO includes a sample config.ini file here:

`${EBROOTBUSCO}/config/config.ini`

Note that all the sample paths are wrong for use with EasyBuild modules
- this may or may not matter/need tweaking.

You can set the config.ini path with:

`$ export BUSCO_CONFIG_FILE="/path/to/myconfig.ini"`

One thing I would definitely change is the path to the busco_downloads
directory, unless you want to fill your home dir with databases...

`download_path = /projects/grp-whatever/busco_downloads/`

...or

`$ busco --download_path=/projects/grp-whatever/busco_downloads/`
